src/components/Dropdown/_Dropdown.scss

```diff
@@ -0,0 +1,80 @@
+.dropdown {
+  color: inherit;
+  display: inline-block;
+
+  &__content {
+    background-color: var(--ui-white);
+    border: 1px solid var(--ui-gray-medium);
+    display: none;
+    padding: var(--space-xxs) var(--space-s) var(--space-xs);
+    position: absolute;
+    transition: all 0.35s;
+    width: 259px;
+    z-index: 1;
+  }
+
+  &__label {
+    @include button--outline;
+
+    align-items: center;
+    border: 1px solid var(--ui-gray-medium);
+    cursor: pointer;
+    font-weight: 300;
+    justify-content: space-between;
+    width: 148px;
+
+    .icon--minus {
+      display: none;
+    }
+
+    &:hover {
+      background-color: var(--ui-gray-light);
+    }
+  }
+
+  &__input {
+    opacity: 0;
+    position: absolute;
+    z-index: -1;
+
+    &:checked {
+      + .dropdown__label .icon--minus {
+        display: inline;
+      }
+
+      + .dropdown__label .icon--plus {
+        display: none;
+      }
+
+      ~ .dropdown__content {
+        background-color: #ffffff;
```

Comment:
Same question as L6. If it's needed, though, let's use the `var(--` associated with white